ð6\venv/lib64/python3.6/site-packages/botocore/vendored/requests/packages/urllib3/exceptions.py 2pyXªh